index.html

```diff
@@ -277,6 +268,28 @@ <h4 class="media-text">LET’S CONNECT</h4>
             </div>
          </div>
         </section>
+        <section class="contact-section">
+          <div class="contact-container">
+            
+            <div class="c-title-container">
+              <h2 class="c-title">Contact me</h2>
+            </div>
+            <div class="c-desc-container">
+              <p class="c-desc">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            </div>
+            <div class="form-container">
+              <form class="contact-form" action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] Great job here 💯, but you should Create a free account on [Formspree](https://formspree.io/) and choose New Form then Update your form's action. 
You have to replace "{form_id}" with the unique "form_id" which you will get after creating the account and form on [Formspree](https://formspree.io/) 👌